parser.go

```diff
@@ -152,10 +164,20 @@ func parserFor(domain string) IParser {
 	return &DefaultParser
 }
 
-func fillIfFound(field *string, re *regexp.Regexp, text *string) {
+func fillIfFound(field *string, re *regexp.Regexp, text *string, skipWordList []string) {
 	if re != nil {
 		if val, found := findAndJoinStrings(text, re); found {
-			*field = val
+			ok := true
```

Comment:
@pokeroklife it's not obvious what `ok` means here. Here we do check if the check should be skipped. So maybe `skip` variable name is more appropriate?